packages/sit-onyx/src/components/OnyxIconButton/OnyxIconButton.vue

```diff
@@ -25,7 +25,7 @@ const emit = defineEmits<{
 
 <template>
   <button
-    class="onyx-icon-button"
+    class="onyx-icon-button onyx-reset"
```

Comment:
Why is this needed?

Reply:
Removed it. My first approach was to set a class.